README.md

```diff
@@ -185,7 +186,6 @@ group.background {
 group.utility {
     // Run on utility queue, in parallel to the previous block
 }
-group.wait()
```

Comment:
I like the non-blocking alternative. However, it feels that the `notify()` method does not follow the convention of the rest of the framework, where the method name indicates the queue (`background`, `utility`, etc.).